### ISSUE TITLE
Sorting by folders

### ISSUE THREAD

**Issue by susl16c** [navidrome/navidrome#1917]
Tell me I have one folder and there are 100 music files in it, your script showed as if there were 100 albums in one folder. I would like to see one album in the albums as it is and there should be 100 music files in it.

**Comment by K4r0qtuYNE5G4qgZ** [comment]
Navidrome only reads Metadata (ID3 tags etc.) so I suppose those 100 files have 100 different albums set and that's why they appear as 100 Albums.

To fix it use a Tool like MP3tag, select all Files and set the same Album for all 100 Files, then they will appear as one Album. I also suggest that you then use the TrackNumberAssistant of MP3Tag to number them from 1-100 because else they gonna have all 0, none or mixed numbers.

I also btw. noticed that Navidrome crashes when there are too many (around ~2400) files in one Album, so don't put too many in one Album, a good max number is ~500 per Album, slows a bit down loading but doesn't crash.

**Comment by susl16c** [comment]
I think this issue could be solved simply by filtering by folders, you can always sort songs by folders, but I didn't find this in the script.
In general, if you bring the script closer to the synology player and if you also make it possible for it to broadcast music via dlna, this would be an ideal script.

**Comment by susl16c** [comment]
Tell me where it is and how to use it did not find such functionality in the script ?

**Comment by susl16c** [comment]
Of course, this utility is good, but when you have 1-2 songs, it will take a lot of time to run them all with the utility, and another question is if it would be possible to display music in folders in the settings, it would make the work easier.

**Comment by K4r0qtuYNE5G4qgZ** [comment]
> Of course, this utility is good, but when you have 1-2 songs, it will take a lot of time to run them all with the utility, and another question is if it would be possible to display music in folders in the settings, it would make the work easier.

Re-Tagging your 100 Files with the same Album set would only take ~1 Minute.

1. Open Folder with the 100 Files in it with MP3Tag
2. CTRL-A to select all Files
3. Set Album Artist to "Various Artists"
4. Set Album to "The Susl16c Compilation"
5. Click Save

And all 100 Files will be and show as in the same Album/Compilation in Navidrome.

**Comment by jrwren** [comment]
I read this issue as a different request:

Ability to browse the underlying folder structure and play music from that.

It is an interesting request. I'm not sure if it would be a good feature for Navidrome or not.
